pkg/summarizer/summary.go

```diff
@@ -574,6 +604,71 @@ nextPath:
 	}
 }
 
+type tabUpdater struct {
+	lock   sync.Mutex
+	update func(context.Context, *configpb.DashboardTab, *configpb.TestGroup, gridReader) func() (*summarypb.DashboardTabSummary, error)
+}
+
+func tabUpdatePool(poolCtx context.Context, log *logrus.Entry, concurrency int) *tabUpdater {
```

Comment:
Right, so instead of having N dashboards updating 1 tab each, we will now have either:
* 1 dashboard updating N tabs or
* x dashboards updating all their tabs and 1 additional dashboard updating some of its tabs, such there are N tabs being updated